src/main/java/frc/robot/commands/swerve/LockSwerveCommand.java

```diff
@@ -0,0 +1,20 @@
+package frc.robot.commands.swerve;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.kinematics.SwerveModuleState;
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+
+import frc.robot.subsystems.drivetrain.BaseSwerveSubsystem;
+
+public class LockSwerveCommand extends InstantCommand {
+    public LockSwerveCommand(BaseSwerveSubsystem swerveSubsystem) {
+        super(() -> {
+            swerveSubsystem.setSwerveModuleStates(
+                new SwerveModuleState(0.0, new Rotation2d(Math.PI / 4.0)),
+                new SwerveModuleState(0.0, new Rotation2d(-Math.PI / 4.0)),
+                new SwerveModuleState(0.0, new Rotation2d(-Math.PI / 4.0)),
+                new SwerveModuleState(0.0, new Rotation2d(Math.PI / 4.0))
```

Comment:
super nitpicky but this can probably be abstracted away so that we don't need a copy here and in (Abstract?)SwerveSubsystem